l3kernel/l3text.dtx

```diff
@@ -188,13 +188,12 @@
 %   \end{itemize}
 %
 %  Determining whether non-letter characters at the start of text should count
-%  as the uppercase element is controllable. When \cs{l_text_titlecase_check_letter_bool} is
-%  \texttt{true}, characters which are not letters (category code~$11$) are
+%  as the uppercase element is controllable. When
+%  \cs{l_text_titlecase_check_letter_bool} is \texttt{true}, codepoints which are
```

Comment:
"codepoints" again!

Reply:
Yes, but this is true here: I'm not sure what else i could say

Reply:
Yes sure.  I was just pointing it out.

The basic idea is to  have somewhere clear definitions of how you are using the terms "Unicode character" and "Unicode codepoint" and emphasising the distinction, if there is any!  Also a note that the "Unicode" is normally implied from the context.